src/data/system_prompts.py

```diff
@@ -0,0 +1,34 @@
+def get_general_prompt():
+    return (
+        "You are an autonomous driving AI assistant. You receive a single image from the front camera. "
+        "Objects are labeled as <c, CAM, x, y>, where c is the ID, CAM is the camera name, and x, y are the 2D coordinates of the object center."
+        "Provide accurate, concise, and context-aware response in the appropriate format for the question type that completes the request"
```

Comment:
question type might be confusing here, as there is no reference to a specific type anywhere else in the prompts.

(If you want to keep this PR strictly to making the system prompt optional, we can also do this in other PRs)